appinfo/info.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<info>
+	<id>brute_force_protection</id>
+	<name>Brute Force Protection</name>
+	<description>ownCloud brute force protection app. It hardens ownCloud security by implementing brute force protection.</description>
+	<summary>ownCloud brute force protection app</summary>
+	<licence>AGPL</licence>

```

Comment:
License file in repo is GPLv3 - @karakayasemi  @PVince81 @pmaier1 what was the decission on this?

Reply:
still un-answered -> no approval @pmaier1 @PVince81 @karakayasemi

Reply:
I'd go for GPLv2. Semih has the last word, IMO.

Reply:
Changed as GPLv2

Reply:
technically speaking you have to ask all contributors if they agree on the re-licensing.
@karakayasemi agreed - so do I :heavy_check_mark: